06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(csv_file):
+    data = {}
+    with open(csv_file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                data[name] = max(data.get(name) or 0, score)
+        except ValueError as e:
+            print("Could not parse csv", end="")
+            print(e)
+
+    return data
+
+def show_high_scores(data):
+    print("High Scores:")
+    print("-------------")
+
+    for name in sorted(data):
+        print(name, data[name])
+
+def leader_board(data):
+    print("Leader board:")
+    print("-------------")
+    sorted_data = sorted(data, key=data.get, reverse=True)
+
+    for index, name in enumerate(sorted_data):
```

Comment:
Please rename `index` to `place`, and use the enumerate `start=` argument instead of adding + 1 to the enumerated value.